src/Symfony/Component/Mime/Address.php

```diff
@@ -93,7 +93,7 @@ public function getEncodedName(): string
     /**
      * @param Address|string $address
      */
```

Comment:
```suggestion
```

This `@param` annotation is now completely redundant. Please delete it if it does not provide any additional information.

Reply:
This was just an example. There's more of those annotations.